script.js

```diff
@@ -0,0 +1,9 @@
+// Display the prompt dialogue while the value assigned to `secret` is not equal to "sesame"
+let secret;
+let password = 'abc123';
+do {
+  secret = prompt("What is the secret password?");
```

Comment:
Since we wan't to delete the `secret` variable on https://github.com/nicquintos/nqjstraining/pull/21/files#diff-ed3ee7e0beea2498ff3b8ca85973d122fc6fa3d585d62b5807ec034d0cf076b3R2

This should now become
```suggestion
prompt("What is the secret password?");
```